src/app/components/validation-item-list/ValidationItem.jsx

```diff
@@ -26,8 +26,9 @@ export default class ValidationItem extends Component {
                     onClick={this.preventUserInteractions}
                     onKeyDown={this.preventUserInteractions}
                     className="readonly-checkbox"
+                    tabIndex={-1}
```

Comment:
just curious  why are you removing this element from the default navigation flow, users will not be able to tab on it using the keyboard?

Reply:
This component is a read-only type of checkbox, so the user won't be able to interact with it, just read its state which is set in code based on their password input. As  `onKeyDown={this.preventUserInteractions}` is used it prevents keyboard users from using the space key to edit the state of the checkbox. As a side-effect, this also creates a keyboard trap. As it is non-interactable I change the tab index to -1 as the user should not be able to edit it. However, thinking a bit more perhaps we should just block space key code that way the user should be able to use voice-over technology to still read the checkboxes state. If they choose to.

We have some work upcoming to update the notification bar (on error this appears) I am planning on inserting an aria alert on the error type.  Which would inform the audio only users of the validation error on this screen too. So they aren't reliant on these verification checkboxes.

Reply:
Cool so, doing something like the following seems to do the trick (where e is the event):
```JavaScript
    const spaceKeyCode = 32;
        if (e.keyCode === spaceKeyCode || e.type == "click") {
            e.preventDefault();
        }
```

Reply:
maybe use just `disable` attribute?

Reply:
so as above `readonly="readonly" `should do the job :-)

Reply:
I think Input of type checkboxes don't acknowledge the readonly attribute, however disabled is a good shout.

Reply:
it does as per W3C standard, which is readonly="readonly" on input. I checked it out. https://www.w3.org/TR/2010/WD-html5-20101019/the-input-element.html

Reply:
The W3C standard in that link states: 

> The readonly attribute can also in some cases (e.g. for the Date state, but not the Checkbox state) make an input element immutable.